### ISSUE TITLE
invalid device group owner is specified for iOS devices

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/stf#554]
```
    group: {
      owner: {
        email: 'administrator@fakedomain.com',
        name: 'administrator'
      },
    },
```

**Comment by vdelendik** [comment]
it should be fixed in stf:2.3-beta18+

**Comment by vdelendik** [comment]
reopening as we don't have such variables on device container level (stf):
```
        email: env.STF_ADMIN_EMAIL
      , name: env.STF_ADMIN_NAME
```

**Comment by vdelendik** [comment]
if we can't get this info at runtime from stf-app we have to adjust setup process and provide this info manually

**Comment by ignacionar** [comment]
**Done:** https://github.com/zebrunner/stf/pull/878

Testing steps:

1. Restart device containers
2. Delete cookies / open STF in incognito window
3. Reload page and check the device group owner

**Comment by dhreben** [comment]
Fixed.

```
~/tools/mcloud-agent$ curl -H "Authorization: Bearer e1e5791cca35403a8...........21bfac89e6656d" https://d........................m/api/v1/devices/6d1e6edcd...................4603216/owner | jq .


{
  "email": "a@zebrunner.com",
  "name": "admin"

```

**Comment by vdelendik** [comment]
fixed indirectly in 2.6.4 after registration valid user ownership:
```
curl -H "Authorization: Bearer a50e...2c4" https://de..m/api/v1/devices/e18...ba8/owner
{"email":"admin@zebrunner.com","name":"admin"}
```

**Comment by dhreben** [comment]
Verified

```
curl -H "Authorization: Bearer 236414........53563" https://dem.........m/api/v1/devices/e18f63ddbea1........ba8/owner
{"email":"ad...@ze......com","name":"admin"
```
